gr-digital/lib/costas_loop_cc_impl.h

```diff
@@ -65,7 +96,18 @@ class costas_loop_cc_impl : public costas_loop_cc
      *  \param sample complex sample
      *  \return the phase error
      */
-    float phase_detector_snr_8(gr_complex sample) const; // for 8PSK
+    float phase_detector_snr_8(gr_complex sample) const // for 8PSK
+    {
+        const float K = (sqrtf(2.0) - 1);
+        const float snr = (sample * std::conj(sample)).real() / d_noise;
```

Comment:
Why the change from ``std::norm()`` to ``x * std::conj(x)``? (Also in ``snr_4`` and ``snr_2``)

Reply:
ping on this query

Reply:
If you are wondering which changes Micheal is referring to (they seem to be buried by now):
https://github.com/gnuradio/gnuradio/pull/3076#discussion_r370948883